### ISSUE TITLE
Deploy is failing

### ISSUE THREAD

**Issue by shunkakinoki** [amondnet/vercel-action#28]
Thanks for the awesome action! 
seriously one of my most beloved github actions out there.

The deploy started failing about several hours ago, perhaps an API change on vercel's part?

```
Error! An unexpected error occurred in inspect: TypeError: Cannot read property 'split' of undefined
```

**Comment by johnmurphy01** [comment]
Having the same issue when deploying my branch deploy

**Comment by facosta0787** [comment]
I'm getting the same error
![image](https://user-images.githubusercontent.com/17883219/89678962-ba082e80-d8b5-11ea-998b-4dd266b1b78c.png)

but the weird part is that the deployment in vercel finishes succesfully.

![image](https://user-images.githubusercontent.com/17883219/89679025-dd32de00-d8b5-11ea-9d6b-86dbb30de035.png)

**Comment by lukasholzer** [comment]
Same here:

![image](https://user-images.githubusercontent.com/11156362/89693951-6840c200-d910-11ea-8c47-75fcd2ae1969.png)

**Comment by arpanlaha** [comment]
It seems to be [this line](https://github.com/amondnet/vercel-action/blob/98fe494a074f14b903db8543c29c51d434cbd4f0/index.js#L24) that's causing the issue, as it calls split on `alias-domains` even though `alias-domains` is intended to be optional.

**Comment by lucis** [comment]
Same here! Adding that, for the same project, it worked a couple of minutes ago

**Comment by amondnet** [comment]
closed by #29

**Comment by holylander** [comment]
thanks for this :)
I noticed that using the action param `vercel-project-name` solved the issue
